impl/src/main/java/org/ehcache/loaderwriter/writebehind/WriteBehindDecoratorLoaderWriterProviderFactory.java

```diff
@@ -50,6 +50,9 @@ public void start(ServiceConfiguration<?> config, ServiceProvider serviceProvide
       
       @Override
       public <K, V> WriteBehindDecoratorLoaderWriter<K, V> createWriteBehindDecoratorLoaderWriter(CacheLoaderWriter<K, V> cacheLoaderWriter, WriteBehindConfiguration configuration) {
+        if (cacheLoaderWriter == null) {
+          throw new RuntimeException("WriteBehind requires non null CacheLoaderWriter.");
```

Comment:
A `NullPointerException` here is better than a simple runtime one - the message being the added information.